govtool/frontend/src/components/molecules/GovernanceActionNewConstitutionDetailsTabContent.tsx

```diff
@@ -0,0 +1,30 @@
+import { Box } from "@mui/material";
+
+import { NewConstitutionAnchor, ProposalData } from "@/models";
+import { useScreenDimension } from "@/hooks";
+
+import { GovernanceActionCardElement } from "./GovernanceActionCardElement";
+
+export const GovernanceActionNewConstitutionDetailsTabContent = ({
+  details,
+}: Pick<ProposalData, "details">) => {
+  const { screenWidth } = useScreenDimension();
+  return (
+    <Box>
+      <GovernanceActionCardElement
+        isCopyButton
+        label="Data Hash"
```

Comment:
```suggestion
        label="New Constitution Hash"
```
a better name for users